api-gateway/src/test/java/com/petclinic/bffapigateway/domainclientlayer/InventoryServiceClientIntegrationTest.java

```diff
@@ -52,6 +57,9 @@ static void tearDown() throws IOException{
         mockWebServer.shutdown();
     }
 
+    InputStream inputStream = getClass().getResourceAsStream("/images/DiagnosticKitImage.jpg");
+    byte[] diagnosticKitImage = ImageUtil.readImage(inputStream);
+
```

Comment:
Good choice of testing the image byte array with an actual pre-determined image.